lib/modules/ai_bot/commands/image_command.rb

```diff
@@ -65,6 +77,8 @@ def process(prompt:)
         end
       end
 
+      return { prompt: prompt, error: "Something went wrong, could not generate image" } if !results
```

Comment:
Error maybe should be `I18n` if it's shown to users?

Reply:
not shown to user just llm